src/GovUk.Education.ExploreEducationStatistics.Common/Model/Chart/Charts.cs

```diff
@@ -79,18 +80,24 @@ public class VerticalBarChart : Chart
 
     public class MapChart : Chart
     {
-        public override ChartType Type => Map;
+        public override ChartType Type => ChartType.Map;
 
         // TODO EES-3319 - make mandatory when all Map Charts are migrated to have a Boundary Level set
         public long? BoundaryLevel { get; set; }
 
+        // Deprecated - Migrate to `DataSetConfigs` planned in EES-4271
         [JsonConverter(typeof(StringEnumConverter))]
         public ChartDataClassification? DataClassification { get; set; }
 
+        // Deprecated - Migration to `DataSetConfigs` planned in EES-4271
         public int? DataGroups { get; set; }
 
-        public List<ChartCustomDataGroup>? CustomDataGroups { get; set; }
+        public MapChartConfig Map { get; set; }
```

Comment:
Should be `} = null!;` if `Map` is not optional and deliberately being initialised as null.

Could consider whether this should be made optional, or initialised with `new()` instead.

Reply:
I believe it should always be initiated, even if the `List<ChartDataSetConfig>` is empty, so I've gone with `new()`.